DemoPage/__tests__/DemoPage-test.js

```diff
@@ -1,18 +1,12 @@
 'use strict';
 
+import { shallow } from 'enzyme'
 import Page from '../'
 
 describe('Demo Page', function() {
-  describe('default', function() {
-    var page;
-
-    beforeEach(function() {
-      page = renderIntoDocument(Page)
-    });
-
-    it('should render Page', function() {
-      page.should.exist;
-    });
+  it('should render Page', function() {
+    var page = shallow(Page)
+    page.should.exist;
```

Comment:
That's fair. For context, the test would flicker by exceeding the default mocha 2000ms timeout. I suspected something deep in the tree was being rendered and causing issues, thus thought a shallow render might be the cure here.